.gitignore

```diff
@@ -94,3 +94,4 @@ logs-debug/
 
 docs/**/log/*
 docs/**/test.sh.run.log
+/.venv/
```

Comment:
Nit: we already have `**/venv*/`. I normally just create `venv` in the root dir.